crates/sncast/tests/e2e/account/create.rs

```diff
@@ -53,6 +53,32 @@ pub async fn test_happy_case() {
     assert!(contents.contains("class_hash"));
 }
 
+#[tokio::test]
+pub async fn test_no_class_hash() {
+    let temp_dir = tempdir().expect("Unable to create a temporary directory");
+    let accounts_file = "accounts.json";
+
+    let args = vec![
+        "--url",
+        URL,
+        "--accounts-file",
+        accounts_file,
+        "account",
+        "create",
+        "--name",
+        "my_account_create_happy",
+        "--salt",
+        "0x1",
+    ];
+
+    let snapbox = runner(&args).current_dir(temp_dir.path());
+
+    snapbox.assert().stderr_matches(indoc! {r"
+        command: account create
+        error: The class 0x58d97f7d76e78f44905cc30cb65b91ea49a4b908a76703c54197bca90f81773 is undeclared, try using --class-hash with a class hash that is already declared
```

Comment:
nit: Maybe instead of using the default value here, we can pass a class hash that is more clearly incorrect, something like `0x123`?

Reply:
https://github.com/foundry-rs/starknet-foundry/pull/1774/commits/2d5d3b6e1662824c5632dc72a48e7eb6fdf179bd